packages/vite/migrations.json

```diff
@@ -23,6 +23,12 @@
       "version": "16.0.0-beta.1",
       "description": "Replace @nrwl/vite with @nx/vite",
       "implementation": "./src/migrations/update-16-0-0-add-nx-packages/update-16-0-0-add-nx-packages"
+    },
+    "update-16-4-0-test-file-config": {
+      "version": "16.4.0-beta.2",
```

Comment:
you'll want to update this to [16.4.0-beta.5](https://www.npmjs.com/package/nx/v/16.4.0-beta.5)